app/assets/stylesheets/pages/home/_ukm.scss

```diff
@@ -0,0 +1,93 @@
+.ukm{
+	margin-top: 33px;
+
+	&__title{
+		font-size: 24px;
+		text-transform: uppercase;
+		width: 400px;
+		color: #484848;
+		line-height: 1.8;
+		margin-bottom: 0;
+		text-align: center;
+	   	margin: auto;
+	}
+
+	&__item-wrap{
+		display: flex;
+		justify-content: space-between;
+		flex-wrap: wrap;
+		margin-top: 55px;
+	}
+
+	&__item{
+		width: 305px;
```

Comment:
don't define it with pixel, use percentage

Reply:
okay, im already change this code with width 29.9%